### PR TITLE
Updates for MinValue and MaxValue fix in cql-execution

### PR DESCRIPTION
**Pull request by losborne**
Point to master branch of cql-execution to include MaxValue/MinValue fix for https://oncprojectracking.healthit.gov/support/projects/BONNIE/issues/BONNIE-521?filter=allopenissues

Add script that allows you to point to a branch of cql-execution (needed since branches don't build the coffeescript by default).

Another script was added to validate the model generation (similar to the validate_dist.sh that already exists).  This is run by travis, which eliminates the need for doing the manual check as part of the PR review process.

Pull requests into cqm-models require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1902
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] If there were any JavaScript changes, this PR has updated the `dist` directory using `npm run dist`
- [x] If applicable, the library version number in `package.json` and `cqm-models.gemspec` has been updated
- [x] All changes can be reproduced by running the generator script
- [x] Cqm-execution fixtures have been updated with the update_cqm_execution_fixtures.sh script inside server-scripts using this branch in the cqm-converter

**Bonnie Reviewer:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code
- [x] You have executed the generator script, and made sure no changes were made that the generator did not reproduce itself

**Cypress Reviewer:**

Name: @mayerm94
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code
- [x] You have executed the generator script, and made sure no changes were made that the generator did not reproduce itself

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -7,9 +7,7 @@ Pull requests into cqm-models require the following. Submitter and reviewer shou
 - [ ] Code diff has been done and been reviewed
 - [ ] Tests are included and test edge cases
 - [ ] Tests have been run locally and pass
-- [ ] If there were any JavaScript changes, this PR has updated the `dist` directory using `npm run dist`
```
Comment (okeefm) [inline .github/PULL_REQUEST_TEMPLATE.md]: Is there a reason why these were removed from the PR template? Are these no longer relevant/do we want to do this in the same PR as other changes?

Comment (losborne) [reply .github/PULL_REQUEST_TEMPLATE.md]: The reasoning was that they are checked automatically by CI/Travis so they don't need to be checkboxes any longer.
